[IMP] accounting: vendor bill digitization & update PO matching (#14332)
odoo/documentation: content/applications/finance/accounting/vendor_bills/invoice_digitization.rst

     <taxes/list_activation>`; this field is not recognized by OCR for the :doc:`Indian
     localization <../../fiscal_localizations/india>`), :guilabel:`Untaxed Amount`, and
     :guilabel:`Total`.

.. _accounting/bill-digitization/vendor-bills-matching-po:

Purchase matching
=================

When a digitized vendor bill is recognized by :abbr:`OCR (optical character recognition)`, Odoo
searches the database for a matching purchase order. If found, the vendor bill can be manually
matched with the existing open purchase order lines.

Once a vendor bill has been :ref:`uploaded <accounting/bill-digitization/vendor-bills-upload>` and
:ref:`digitized <accounting/bill-digitization/digitization>`, click the :guilabel:`Purchase
matching` smart button to access the :guilabel:`Purchase matching` list view, displaying all
purchase orders linked to the corresponding vendor. Then, select the relevant lines from the
purchase order and the draft vendor bill (shown in grey), and click :guilabel:`Match`.

.. tip::
   In the :guilabel:`Purchase Matching` list view, update the :guilabel:`Quantity` and
   :guilabel:`Price` in the purchase order lines, if necessary.

> If there is no existing purchase order related to the uploaded vendor bill, a new purchase order

Reviewer: ```suggestion
If there is no existing purchase order related to the vendor of the uploaded vendor bill, a new purchase order
```

can be directly created from the vendor bill lines. To do so, follow these steps:

> #. Once the vendor bill is uploaded, make sure the :guilabel:`Vendor` field is filled in.

Reviewer: ```suggestion
#. Once the vendor bill is uploaded, make sure the :guilabel:`Vendor` field is filled in with the correct vendor.
```
(to be improved)

#. Click the :guilabel:`Purchase matching` smart button, select the corresponding draft vendor bill
   (shown in grey), and click :guilabel:`Add to PO`.
#. In the :guilabel:`Add to Purchase Order` window, start typing in the :guilabel:`Purchase Order`
   field and select :guilabel:`Create and edit`.

> #. In the :guilabel:`Create Purchase Order` window, complete all :ref:`required fields
>    <purchase/manage_deals/create-new-rfq>` in the purchase order form and click :guilabel:`Confirm

Reviewer: ```suggestion
#. In the :guilabel:`Create Purchase Order` window, select the vendor from the bill, then complete all :ref:`required fields <purchase/manage_deals/create-new-rfq>` and click :guilabel:`Confirm
```
(feel free to improve this! I removed purchase order form since you said "in the Create purchase order window" at the beginning of the sentence)

   order`.

> #. In the :guilabel:`Purchase Matching` list view, select the relevant lines from the purchase order
>    and the lines from the draft vendor bill (shown in grey), and click :guilabel:`Match`.

Reviewer: ```suggestion
#. In the :guilabel:`Purchase Matching` list view, select the relevant purchase order lines and the draft vendor bill (shown in grey), and click :guilabel:`Match`.
```
since there's only one line for the vendor bill?

.. Note::
   If any information required for the purchase order fields is missing, click :guilabel:`Save and
   Close` in the :guilabel:`Create Purchase Order` window. Then, open the Purchase app to fill in

>    the fields and confirm the purchase order.

Reviewer 2: I added a link to the "confirm order" section of the RFQ page because I thought that the Purchase page could be confusing. Or I can add both? What do you think?
